src/dotnet/APIView/APIViewWeb/Languages/PythonLanguageService.cs

```diff
@@ -2,10 +2,10 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Diagnostics;
+using System.Diagnostics;
```

Comment:
I only intended to change the version in this file but git kept seeing a diff. I suspect it is due to line endings. @praveenkuttappan please let me know what you would like me to do.